### PR TITLE
♻️ Optimize syntax and beautify the code

### PR DESCRIPTION
**Pull request by patrikspiess**
- Update docstrings
    - No more types in Args: and Returns: because they are already in function/method definition
    - Aligning texts
- Update Logs
    - Surround %s with ' so they are printed in different color
    - Start all log and error messages with Uppercae
- Remove unused blank lines
- Add blank lines after indented blocks
- Change typing Union[xyz, None] to Optional[xyz]
- Other simple changes

### CODE REVIEW

File: fotoobo/tools/fmg/get.py [hunk 1]
```diff
@@ -18,25 +17,21 @@ def adoms(host: str) -> Result[str]:
...
     """
     inventory = Inventory(config.inventory_file)
     fmg = inventory.get_item(host, "fortimanager")
     result = Result[str]()
-
     log.debug("FortiManager get adoms ...")
-
     fmg.login()
     fmg_adoms = fmg.get_adoms()
-
```
Comment (lucmurer) [inline fotoobo/tools/fmg/get.py:34]: dito

File: fotoobo/tools/fmg/get.py [hunk 2]
```diff
@@ -123,7 +127,6 @@ def policy(
     }
     result = fmg.api("post", payload=payload, timeout=30)
     data = result.json()
-
```
Comment (lucmurer) [inline fotoobo/tools/fmg/get.py:126]: dito

File: fotoobo/tools/get.py [hunk 3]
```diff
@@ -37,21 +38,19 @@ def version(verbose: bool = False) -> Result[List[Dict[str, str]]]:
...
     Args:
-        verbose:    Whether we want verbose output
+        verbose: Whether we want verbose output
     """
-    log.debug("printing fotoobo version information: %s", __version__)
+    log.debug("Print fotoobo version information: '%s'", __version__)
 
     result = Result[List[Dict[str, str]]]()
     versions = [{"module": "fotoobo", "version": __version__}]
-
```
Comment (lucmurer) [inline fotoobo/tools/get.py:46]: dito

**Review by lucmurer — CHANGES_REQUESTED** [review]
Very good an thorough cleanup of the code!!

For better legibility I propose to also change the following:
 - Every Python keyword that results in a following block (`if`, `for`, `with`, ...) should be prepended with a blank line, except
     - If the line before is a line comment describing the condition or block
     - If the preceding line is solely for defining a variable that gets used/filled in the block

I would apply the same rules for `return` statements.

I pointed out quite a lot of spots that had a direct change, I saw there are more places in the code which would need cleanup based on my proposal.

If you agree with this proposal please update the pull request again, otherwise I will look again at the pull request and approve it.

**Comment by patrikspiess** [comment]
> Very good an thorough cleanup of the code!!
> 
> For better legibility I propose to also change the following:
> 
> * Every Python keyword that results in a following block (`if`, `for`, `with`, ...) should be prepended with a blank line, except
>   
>   * If the line before is a line comment describing the condition or block
>   * If the preceding line is solely for defining a variable that gets used/filled in the block
> 
> I would apply the same rules for `return` statements.
> 
> I pointed out quite a lot of spots that had a direct change, I saw there are more places in the code which would need cleanup based on my proposal.
> 
> If you agree with this proposal please update the pull request again, otherwise I will look again at the pull request and approve it.

Thank you. I went over all the files and added/removed blank lines as by your suggestion.

File: fotoobo/fortinet/fortigate_config_check.py [hunk 4]
```diff
@@ -32,8 +32,8 @@ def add_message(self, chk: Dict[str, Any], msg: str) -> None:
         Generates a styled message and appends it to the results.
 
         Args:
-            chk (Dict[str, any]): the check which generated the message
-            msg (str): the message to sent to the messages list
+            chk: The check which generated the message
+            msg: The message to sent to the messages list
```
Comment (lucmurer) [inline fotoobo/fortinet/fortigate_config_check.py:36]: Typo here: "to sen**d** to"

File: fotoobo/fortinet/fortimanager.py [hunk 5]
```diff
@@ -24,13 +24,13 @@ def __init__(self, hostname: str, username: str, password: str, **kwargs: Any) -
         Set some initial parameters.
 
         Args:
-            hostname:       The hostname of the FortiGate to connect to
-            username:       The Username
-            password:       The password
+            hostname: The hostname of the FortiGate to connect to
+            username: The Username
```
Comment (lucmurer) [inline fotoobo/fortinet/fortimanager.py:28]: typo: **u**sername

File: fotoobo/tools/fmg/main.py [hunk 6]
```diff
@@ -62,22 +64,24 @@ def post(file: Path, adom: str, host: str) -> Result[str]:
     POST the given configuration from a JSON file to the FortiManager
 
     Args:
-        file:   The configuration file to oad the configuration from
-        adom:   The ADOM to assign the global policy to
-        host:   The FortiManager defined in inventory
+        file: The configuration file to oad the configuration from
```
Comment (lucmurer) [inline fotoobo/tools/fmg/main.py:67]: I think "oad" is also a typo here. Should this be "add"?

**Review by lucmurer — CHANGES_REQUESTED** [review]
Good job, looks very nice now! 
Just a few typos remaining :-)

**Pull request merged**